bids/layout/config/bids-nodot.json

```diff
@@ -12,6 +12,10 @@
             "mandatory": false,
             "directory": "{subject}{session}"
         },
+        {
```

Comment:
@Remi-Gau Sorry to do this to you, but I think we no longer support `bids-nodot`. It probably should have been removed when we finished the transition to `extension_initial_dot` defaulting to `True`.

Reply:
No worries, search and replace are easy enough in vscode. :-)

But should I just nuke the file though? :boom:

Reply:
Fine with me. Would be good to make sure that we don't have any tests or code-paths still depending on it.

Reply:
is the coverage good enough that it will be detected? Should I just try? if so maybe better in another PR, no?

Reply:
I'm not positive. Happy to push to another PR.